packages/graphback-core/src/plugin/GraphbackCRUDGeneratorConfig.ts

```diff
@@ -1,21 +1,22 @@
 /**
- * Configuration for global values for graphback 
+ * Configuration for global values for graphback
  */
 export interface GraphbackCRUDGeneratorConfig {
-    //Generate create method
-    create?: boolean;
-    //Generate update method
-    update?: boolean;
-    //tslint:disable-next-line
-    delete?: boolean;
-    //Generate find method
-    find?: boolean;
-    //Generate findAll method
-    findAll?: boolean;
-    //Generate subscription for create operation
-    subCreate?: boolean;
-    //Generate subscription for update operation
-    subUpdate?: boolean;
-    //Generate subscription for delete operation
-    subDelete?: boolean;
+  //Generate create method
+  create?: boolean;
+  //Generate update method
+  update?: boolean;
+  // Generate delete method
+  delete?: boolean;
+  //Generate findOne method
+  findOne?: boolean;
+  findAll?: boolean;
+  //Generate find method
+  find?: boolean;
```

Comment:
Potentially silly question. If `findOne` fetches one object based on some filter then what's the difference between `find` and `findall`?

Reply:
findAll is going away, I just kept it in for now to limit the amount of things that break.

Reply:
find now does the exact same.

Reply:
I would kill it. Ideally we want next release to be uber breaking so people do stuff once